.circleci/config.yml

```diff
@@ -162,6 +162,7 @@ jobs:
                   key: v0.5-repository_consistency-{{ checksum "setup.py" }}
                   paths:
                       - '~/.cache/pip'
+            - run: pip install pytest timeout_decorator parameterized
```

Comment:
If we want to use the changes in `check_repo.py`, we need more libs being installed. This is a disadvantage of this new approach.